src/Spago/Config.purs

```diff
@@ -23,14 +23,19 @@ import Affjax.Node as Http
 import Affjax.ResponseFormat as Response
 import Affjax.StatusCode (StatusCode(..))
 import Data.Array as Array
+import Data.CodePoint.Unicode as Unicode
 import Data.Codec.Argonaut.Record as CAR
+import Data.Enum (fromEnum)
 import Data.Foldable as Foldable
 import Data.HTTP.Method as Method
+import Data.Int (hexadecimal)
```

Comment:
Could we qualify all of these function imports? We generally import directly only types, everything else is qualified.